EESSI-pilot-install-software.sh

```diff
@@ -70,16 +70,15 @@ if [[ "$1" == "--generic" || "$EASYBUILD_OPTARCH" == "GENERIC" ]]; then
 fi
 
 
-# make sure that $PATH starts with $CVMFS_REPO
-# (if not, we're not running in the environment set up by 'startprefix')
-if [[ $PATH = ${CVMFS_REPO}* ]]; then
+# make sure we're in Prefix environment by which path to 'bash' command
+if [[ $(which bash) = ${EPREFIX}/bin/bash ]]; then
     echo_green ">> It looks like we're in a Gentoo Prefix environment, good!"
 else
     error "Not running in Gentoo Prefix environment, run '${EPREFIX}/startprefix' first!"
 fi
 
 echo ">> Initializing Lmod..."
-source $EPREFIX/usr/lmod/lmod/init/bash
+source $EPREFIX/usr/share/Lmod/init/bash
```

Comment:
This update is required for bash initialisation as well (also for Magic Castle), will I open a separate PR for that?

Reply:
Yes, let's do init script for `2021.02` in a separate PR.